docs/services/robot-service.md

```diff
@@ -3,11 +3,12 @@ title: "Robot Service"
 linkTitle: "Robot"
 description: "The Robot Service consists of a set of robot monitoring and management APIs that most robots support."
 type: "docs"
-draft: "true"
+draft: "false"
 weight: 10
 tags: ["robot state", "services"]
 # SME: Cheuk
 ---
+
 Robot Service constitutes a minimal set of APIs that most robots (Viam Server, [Viam Python SDK](https://python.viam.dev/) and various SDKs) should support.
```

Comment:
```suggestion
The Robot Service refers to the minimum set of APIs that most robots running `viam-server` support.
This means that any of the various robot clients in Viam, like the Python SDK, Go Client SDK, or modular resources, should also support methods to interface with these same APIs from the client (instead of server) side.
If you are implementing your own client, keep this in mind.
```

Reply:
Oh oops I'm sorry-- I did not mean to include this file in this PR. I think it should stay as a draft and not be reviewed for now. My thought with this PR was to just move the method documentation that you already made on the APIs page to a different page for consistency, but not to address this page yet.